README.md

```diff
@@ -25,22 +25,7 @@ This project provides an API gateway in front of other Democracy Club APIs.
 
 ### Documentation
 
-The API docs use [drafter 3](https://github.com/apiaryio/drafter/) for parsing [API Blueprint](https://apiblueprint.org/).
-
-This needs to be installed/compiled manually:
```

Comment:
Could you include the same non-linux comment as you've put on https://github.com/DemocracyClub/UK-Polling-Stations/pull/2650/files#diff-04c6e90faac2675aa89e2176d2eec7d8R39?